scripts/Costs.py

```diff
@@ -533,42 +462,6 @@ def getUsaVariableCost():
             mode = 2
             outData.append([region,tech,mode,year,value])
 
-    #Get trade costs

```

Comment:
The Variable Costs section in the Wiki is currently empty, and so could use updating.

Reply:
created issue